README.md

```diff
@@ -1,5 +1,5 @@
 # Qtum adapter to Ethereum JSON RPC
-Janus is and old school ETH web3 HTTP provider that translates Ethereum JSON RPC calls into their equivalent Qtum RPC call/s. The current version self hosts the keys and does not support web sockets.
+Janus is and old school ETH web3 HTTP provider that translates Ethereum JSON RPC calls into their equivalent Qtum RPC call/s. The current version self hosts the keys and supports web sockets.
```

Comment:
With this we can get rid of old school and HTTP from this description. Something like "Janus is a web3 proxy adapter that can be used as a web3 provider to interact with Qtum. It supports HTTP and websockets and the current version enables self hosting of keys."

Reply:
Updated